packages/core/middleware/api-extractor.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../../api-extractor.base.json",
+  "mainEntryPointFilePath": "./lib/index.d.ts",
+  "dtsRollup": {
+    "untrimmedFilePath": "./lib/<unscopedPackageName>.d.ts"
+  },
+  "docModel": {
+    "apiJsonFilePath": "<projectFolder>/core/docs/core/api-reference/<unscopedPackageName>.api.json"
+  }
+}
```

Missing newline at the end of the file.

```suggestion
}

```